dubbo-common/src/main/java/org/apache/dubbo/common/deploy/DeployListener.java

```diff
@@ -28,6 +28,8 @@ public interface DeployListener<E extends ScopeModel> {
 
     void onStarted(E scopeModel);
 
+    void onCompletion(E scopeModel);
+
     void onStopping(E scopeModel);
 
     void onStopped(E scopeModel);
```

Comment:
I think we should add some comment on these statuses.

Reply:
Okay, I‘ll add some comment.